src/ground-truth.rkt

```diff
@@ -66,4 +65,9 @@
       (make-sampler ctx pre fn)))
   (timeline-event! 'sample)
   (match-define (cons table2 results) (batch-prepare-points fn ctx sampler))
-  (cons (combine-tables table table2) results))
+  (representation-repr->bf (context-repr ctx))
+  (define combined-table (combine-tables table table2))
+  (when (> (hash-ref combined-table 'infinite 0.0) 0.2)
+    (warn 'inf-points #:url "faq.html#inf-points"
+      "Getting infinite outputs for ~a of points. You may want to add a precondition." (hash-ref combined-table 'infinite)))
```

Comment:
Yes, please format this as an actual percentage. You can use `(format-accuracy percentage 1 #:unit "%")` perhaps?